src/Open3D/Visualization/Rendering/Filament/FilamentResourceManager.cpp

```diff
@@ -99,10 +99,16 @@ FilamentResourceManager::FilamentResourceManager(filament::Engine& aEngine)
     : engine_(aEngine) {
     // FIXME: Move to precompiled resource blobs
     const std::string resourceRoot = gui::Application::GetInstance().GetResourcePath();
+
     const auto depthPath = resourceRoot + "/depth.filamat";
     const auto hDepth = CreateMaterial(ResourceLoadRequest(depthPath.data()));
     auto depthMat = materials_[hDepth];
     materialInstances_[kDepthMaterial] = std::move(MakeShared(depthMat->createInstance(), engine_));
+
+    const auto normalsPath = resourceRoot + "/normals.filamat";
+    const auto hNormals = CreateMaterial(ResourceLoadRequest(normalsPath.data()));
+    auto normalsMat = materials_[hNormals];
+    materialInstances_[kNormalsMaterial] = std::move(MakeShared(normalsMat->createInstance(), engine_));
```

Comment:
Lines 106 and 111 don't compile with Clang, "std::move prevents copy elision". Since materialInstances_ is a vector of `shared_ptr`s they should be copied (but apparently the compiled just constructs it in place, judging from the error). (Was materialInstances_ a vectory of `unique_ptr` at some point?)

Reply:
Yeap, they was. Will fix it